app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -66,6 +65,7 @@ public void onStart() {
         loginFragmentViewModel.getLogin().observe(this, login -> {
             binding.setLogin(login);
         });
+        sharedViewModel.getEmail().observe(this, email -> binding.getLogin().setEmail(email));
```

Comment:
I will suggest moving it after line 66 inside the callback if possible. We should only call this once we are sure that the  `login` variable is set on the binding.